test/cloudtest/pkg/tests/shell_provider_test.go

```diff
@@ -291,7 +292,7 @@ func TestUsedClusterCancel(t *testing.T) {
 	testConfig.ConfigRoot = tmpDir
 	createProvider(testConfig, "a_provider")
 	p2 := createProvider(testConfig, "b_provider")
-	p2.TestDelay = 15
+	p2.TestDelay = 5
```

Comment:
why it is 5? Can we use some lower value?

Reply:
To test a_provider destruction (upon completion of assigned tests, not the whole testing) all tests in a_provider should finish before total tests execution finished (b_provider in out case). Effectively, it means that 'the longest test sequence on a_provider must be less than longest sequence in optimal run on b_provider'

a_provider longest sequence: TestTimeout(5sec) + {TestPass|TesFail}

b_provider longest sequence in 'fastest run' = max(
   TestPass + b_provider.TestDelay(5sec) + TestFail
   TestTimeout(5s) // no TestDelay
)// max

so, TestDelay must be > 5 to ensure the test to pass
currently, it's 7 in my ongoing PR